### PR TITLE
get-started edits

### PR DESCRIPTION
**Pull request by DougManuel**
Quite a few edits throughout.

### CODE REVIEW

File: vignettes/get_started.Rmd [hunk 1]
```diff
@@ -1,20 +1,18 @@
 ---
-title: "Using cchsflow"
+title: "Get started"
 output: rmarkdown::html_vignette
+date: "`r Sys.Date()`"
```
Comment (wyusuf068) [inline vignettes/get_started.Rmd]: Should we have a date for this vignette? The other vignettes do not have dates on them.

File: vignettes/get_started.Rmd [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
 ```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(echo = TRUE, collapse = TRUE, comment = "#>")
 ```
 
-## Getting Started
-
-### Loading the necessary packages
-
+## Load packages
```
Comment (wyusuf068) [inline vignettes/get_started.Rmd]: Should we also include load datasets in this section as well?
I was thinking adding pseudocode to instruct users to load the CCHS datasets that are saved on their computer into the environment. Something like this:
`cchs2001 <- read.csv("~/Documents/cchs2001.csv")`

File: vignettes/get_started.Rmd (hunk 1 above)
Comment (DougManuel) [reply vignettes/get_started.Rmd]: let's leave out for now.

File: vignettes/get_started.Rmd (hunk 2 above)
Comment (DougManuel) [reply vignettes/get_started.Rmd]: Good idea.

Comment (DougManuel) [reply vignettes/get_started.Rmd]: I'll add that.

**Pull request merged**